### PR TITLE
Update with Joe's latest draft

### PR DESCRIPTION
**Pull request by zmanian**
Updating the github with Joe's latest draft. [Rendered](https://github.com/iqlusioninc/governance/tree/zaki/optimize-inflation-rate-of-change/proposals/2020-11-inflation-rate-change)

### CODE REVIEW

File: proposals/2020-11-inflation-rate-change/README.md [hunk 1]
```diff
@@ -1,40 +1,30 @@
-﻿
-Cosmos Hub Inflation variable modification: Inflation Rate Change
-
-**Quick Summary of issue**
-
-There are 6 main variables that control the maximum, minimum, & change of the atom inflation rate for the cosmos hub. These variables can be found inside the mint module via the attached hyperlink: [https://github.com/gavinly/CosmosParametersWiki/blob/master/Mint.md](https://github.com/gavinly/CosmosParametersWiki/blob/master/Mint.md)
-
+﻿# Quick Summary of The Issue
+There are 6 main variables that control the maximum, minimum, & change of the atom inflation rate for the cosmos hub. These variables can be found inside the mint module via the attached hyperlink: https://github.com/gavinly/CosmosParametersWiki/blob/master/Mint.md
```
Comment (ebuchman) [inline proposals/2020-11-inflation-rate-change/README.md:2]:
```suggestion
There are 6 main variables that control the maximum, minimum, & change of the atom inflation rate for the cosmos hub. These variables can be found inside the mint module via the attached hyperlink: https://github.com/cosmos/governance/blob/master/params-change/Mint.md

```

File: proposals/2020-11-inflation-rate-change/README.md [hunk 2]
```diff
@@ -1,40 +1,30 @@
...
 In this proposal we will be looking at adjusting the inflation rate change variable.
 
-- When the cosmos hub inflation dynamics were originally designed, the goal was for the hub to go from the minimum rate (7%) to the maximum rate (20%) in roughly one year after a shock had unbond occurred. Thus, for the variable “inflation rate change” a value of .13 was chosen. Unfortunately, in practice the variable didn’t work as intended due to the fact the change in the inflation rate for the hub is proportional to the Target Bonded / current bonded ratio.
-
+When the cosmos hub inflation dynamics were originally designed, the goal was for the hub to go from the minimum rate (7%) to the maximum rate (20%) in roughly one year after a shock unbond had occurred. Thus, for the variable “inflation rate change” a value of .13 was chosen. Unfortunately, in practice the variable didn’t work as intended due to the fact the change in the inflation rate for the hub is proportional to the Target Bonded / current bonded ratio.
+How to select “inflation rate change” AKA Maximum Slope of the inflation curve
```
Comment (ebuchman) [inline proposals/2020-11-inflation-rate-change/README.md:6]:
```suggestion

## How to select “inflation rate change” AKA Maximum Slope of the inflation curve

```

**Review by ebuchman — APPROVED** [review]
Thanks for updating. Noticed there's no link back to here or the forum in the proposal.json. We should probably prioritize getting some tooling built that helps format and submit the gov proposals.

**Comment by hxrts** [comment]
Any idea when this will be submitted?

**Comment by hxrts** [comment]
I think we should probably move this to the gaia repo after we merge the migration PR. Unless this is no longer being proposed to the hub.

**Pull request merged**